src/System.Data.Odbc/ref/System.Data.Odbc.cs

```diff
@@ -96,6 +99,9 @@ public sealed partial class OdbcConnection : System.Data.Common.DbConnection, Sy
         public new System.Data.Odbc.OdbcCommand CreateCommand() { throw null; }
         protected override System.Data.Common.DbCommand CreateDbCommand() { throw null; }
         protected override void Dispose(bool disposing) { }
+        public override System.Data.DataTable GetSchema() { throw null; }
```

Comment:
These APIs were previously not exposed. Should they be? @divega, @ajcvickers, @afsanehr, @david-engel, @tarikulsabbir

Reply:
@ahsonkhan I am not sure what this means. This method was added to the base class (DbConnection) in 2.0. Then in 2.1, https://github.com/dotnet/corefx/commit/9dc5aad6c1db08049f00d01f504a742cc9751199 fixed the fact that it was not overridden in this class.

Reply:
Does it matter for the purpose of the API comparison when a method is overridden in a derived type? It should never be a breaking change to the contract.

Reply:
Since the class is sealed, adding the override here is unnecessary but also harmless.